### PR TITLE
gcp - add ml engine job resource

### PR DESCRIPTION
**Pull request by mediapills**
Added initial classes with a query and get functionality for jobs resources in ML Engine service

### CODE REVIEW

File: tools/c7n_gcp/tests/test_mlengine.py [hunk 1]
```diff
@@ -52,3 +52,43 @@ def test_models_get(self):
         })
 
         self.assertEqual(resource['name'], "projects/cloud-custodian/models/{}".format(name))
+
+
+class TestMLJobs(BaseTest):
```
Comment (jondavidi) [inline tools/c7n_gcp/tests/test_mlengine.py]: should be singular

File: tools/c7n_gcp/c7n_gcp/resources/mlengine.py [hunk 2]
```diff
@@ -32,7 +32,27 @@ class resource_type(TypeInfo):
...
-                    'name': 'projects/{}/models/{}'.format(
-                        resource_info['project_id'],
-                        resource_info['name'].rsplit('/', 1)[-1])})
+                'get', {'name': 'projects/{}/models/{}'.format(
+                    resource_info['project_id'],
+                    resource_info['name'].rsplit('/', 1)[-1])})
+
+
+@resources.register('ml-job')
+class MLJobs(QueryResourceManager):
```
Comment (jondavidi) [inline tools/c7n_gcp/c7n_gcp/resources/mlengine.py]: should be singular

**Comment by mediapills** [comment]
@jondavidi  done

File: tools/c7n_gcp/tests/test_mlengine.py [hunk 3]
```diff
@@ -52,3 +52,43 @@ def test_models_get(self):
         })
 
         self.assertEqual(resource['name'], "projects/cloud-custodian/models/{}".format(name))
+
+
+class TestMLJob(BaseTest):
```
Comment (jondavidi) [inline tools/c7n_gcp/tests/test_mlengine.py]: just noticed this, should be MLJobTest like in other test classes

**Comment by mediapills** [comment]
Pull request [#3835](https://github.com/cloud-custodian/cloud-custodian/pull/3835) into upstream has been created.

**Pull request closed without merging**